doc/tutorials/vcat_configuration.rst

```diff
@@ -0,0 +1,87 @@
+.. _vcat_configuration:
+
+Enable vCAT Configuration
+#########################
+
+vCAT is built on top of RDT, so to use vCAT we must first enable RDT.
+For details on enabling RDT configuration on ACRN, see :ref:`rdt_configuration`.
+For details on ACRN vCAT high-level design, see :ref:`hv_vcat`.
+
+The vCAT feature is disabled by default in ACRN. You can enable it with these settings:
+
+#. Configure system level features:
+
+   - Edit :option:`hv.FEATURES.RDT.RDT_ENABLED` to `y` to enable RDT
+
+   - Edit :option:`hv.FEATURES.RDT.CDP_ENABLED` to `n` to disable CDP.
+     Currently vCAT requires CDP to be disabled.
+
+   - Edit :option:`hv.FEATURES.RDT.VCAT_ENABLED` to `y` to enable vCAT
+
+     .. code-block:: xml
+        :emphasize-lines: 3,4,5
+
+        <FEATURES>
+            <RDT>
+                <RDT_ENABLED>y</RDT_ENABLED>
+                <CDP_ENABLED>n</CDP_ENABLED>
+                <VCAT_ENABLED>y</VCAT_ENABLED>
+                <CLOS_MASK></CLOS_MASK>
+            </RDT>
+        </FEATURES>
+
+#. In each guest VM configuration:
+
+   - Edit :option:`vm.guest_flags.guest_flag` and add ``GUEST_FLAG_VCAT_ENABLED``
```

Comment:
Is that the case here too, i.e. that `GUEST_FLAG_VCAT_ENABLED` is a flag that can be set for the Service VM?

Reply:
yes